docs/source/GETTING_STARTED.rst

```diff
@@ -14,7 +14,7 @@ Google Cloud Setup
 
 .. warning:: Google Cloud Platform must approve several requests that may take up to 1 day to complete.
 
-1. If necessary, create an account at `Google Cloud <https://cloud.google.com>`_ and create a Google Cloud project, making sure you have at least one account with the `Owner` role. Take note of the project ID (you will need this later during Kiosk configuration).
+1. If necessary, create an account at `Google Cloud <https://cloud.google.com>`_ and create a Google Cloud project, making sure you have at least one account with the `Owner` role.
 
 2. Make sure the `Kubernetes Engine API <https://console.cloud.google.com/apis/api/container.googleapis.com/overview>`_ is enabled.
 
```

Comment:
"Google does not include these GPU nodes by default as part of its free tier thus necessitating the upgrade." Thus is problematic here. It should be "free tier; thus, necessitating the upgrade." or "free tier, which is why the upgrade is required."

Reply:
Rereading this section makes we want to reshuffle the order of the sentences:

> 3. The recent success of deep learning has been critically dependent on accelerated hardware like GPUs. Similarly, the strength of the DeepCell Kiosk is its ability to recruit and scale GPU nodes based on demand. In order to add accelerated hardware to the clusters you will launch, you will need to `upgrade <https://cloud.google.com/free/docs/gcp-free-tier#how-to-upgrade>`_ your Google Cloud account.

> NOTE: The account upgrade may take some time, as Google will need to approve the upgrade. You may also need to log in and out of your account for the upgrade to take effect. Once your account is upgraded you should be able to see GPU options in `the quota panel <https://console.cloud.google.com/iam-admin/quotas>`_.

How does that sound?

Reply:
I like that much better than what we had. The only concern I would have (and this is true in both versions) is that the user may think that adding a GPU is optional.